front/public/locales/fr/stdcm.json

```
@@ -10,18 +10,22 @@
   "pathfindingFailed": "Aucun chemin n'a été trouvé pour ces points de jalonnement.",
   "pleaseWait": "Veuillez patientez…",
   "simulation": {
-    "available": "Simulation disponible",
     "averageRequestTime": "Pour votre demande, le temps nécessaire est généralement de 90 secondes.",
     "calculatingSimulation": "Calcul en cours...",
     "getSimulation": "Obtenir la simulation",
+    "modifySearchCriteria": "Vous pouvez modifier vos critères de recherche pour trouver une solution.",
+    "notFound": "Désolé, aucune solution n'a été trouvée.",
     "pendingSimulation": "simulation en cours",
     "results": {
+      "available": "Simulation disponible",
```

As it's not a big change : seems like sketch now use this text (need to change english as well)

```suggestion
      "available": "Calcul terminé",
```

---

Done